### DISCUSSION TITLE
Why is "dict" not "proj"?

### DISCUSSION THREAD

**Discussion by Xiaozizhuo** [PMEAL/porespy#699] [Q&A]
I wanted to use Porespy's snow algorithm to extract the geometric data of the network model and its pores and throat on a fiber pad. But when I follow the steps of the example, I have the following error, why is this?
![TU ](https://user-images.githubusercontent.com/98071787/170812746-bcac67d4-6414-4b99-9a1e-0a4e79932723.png)
And how to import Porespy's data into MATLAB? I looked at Porespy's functional modules and didn't seem to have saved the code to MATLAB.

**Comment by jgostick** [comment]
The ``pn`` variable on line 30 is comging for porespy, and returns a python ``dict``.  You need to add several steps between lines 30 and 31, so that you convert the ``pn`` into an openpnm object, then you can call openpnm.io.VTK.save.  Maybe [this example](https://porespy.org/examples/networks/tutorials/snow_basic.html) will help you?
